addons/gdUnit4/src/network/rpc/RPC.gd

```diff
@@ -18,7 +18,76 @@ func get_data() -> Object:
 
 
 func serialize() -> String:
-	return JSON.stringify(inst_to_dict(self))
+	var data = inst_to_dict(self)
+	_strip_bbcode_from_dict(data)
```

Comment:
Stripping out the bbcode do lose all the colors.
The UI can't restore this info without bbcodes.

Reply:
Thanks Mike. Which panel will be missing colors? I still see colors in the UI -- granted I'm brand new to this stuff and barely know the UI. Here;s a screenshot of what I see:

<img width="502" height="531" alt="image" src="https://github.com/user-attachments/assets/831d32a1-7586-4ef3-8e52-9f31a0043174" />

Reply:
Take a look at the error messages, it is normally colored. After your fix is now gray.

Reply:
Yes, please close this PR and feel free to create a new one with the network fix.
Thanks for creating the very good described bug issue.